### ISSUE TITLE
page-break vs. break

### ISSUE THREAD

**Issue by agamemnus** [dompdf/dompdf#1244]
https://www.w3.org/TR/css-break-3/#page-break-properties

I think that dompdf checks for "page-break" but this actually is just an alias and converts to "break" as described in that link above...

**Comment by bsweeney** [comment]
This is true for CSS3, but as you'll note on the project readme:

> dompdf is (mostly) a CSS 2.1 compliant HTML layout and rendering engine

We're slowly adding support for the CSS3 spec. This one should be easy enough to implement.

**Comment by agamemnus** [comment]
I thought this rule was supposed to be for css2 compliance somehow?

**Comment by bsweeney** [comment]
The wording is a bit confusing. The CSS3 spec defines `break-*`. CSS3 compliant user agents that also conform to CSS2 should alias the `page-break-*` properties. Dompdf is currently aiming for CSS2 compliance enhanced with CSS3 support.

So at some point we'll be CSS3 compliant and will definitely want to do this. I don't think it will have adverse affects to implement sooner rather than later.

**Comment by agamemnus** [comment]
So the CSS3 is actually "break" and not "page-break"?

**Comment by bsweeney** [comment]
You are correct. Chrome must be fixing up the stylesheet to CSS3 conformance rather than translate.
